resources/sparql/taxonomy/ncbi-id.mustache

```diff
@@ -0,0 +1,12 @@
+
+prefix obo: <http://purl.obolibrary.org/obo/>
+
+# Given a bio entity, return the NCBI taxonomy ID to which it belongs.
+select ("{{src-id}}" as ?entity_id) ?ncbi_id
+where {
+  <{{bio-id}}> obo:RO_0002162 ?ncbi_id
```

Comment:
To be safe, I think we should be referencing the restriction that is onProperty RO_0002162 and someValuesFrom ?ncbi_id.  As written, the query works but relies on what appears to be a shortcut relation that we should probably remove from KaBOB.